lab-glen/test/test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Client = require('../model/client');
+const expect = require('chai').expect;
+
+describe('Nickname Test', function() {
```

Comment:
let client = net.connect({port: 3000}, () => {
         client.once('data', function(data) {
           expect(data.toString()).to.include('has connected!')
         })
         client.end()
         done()
hopefully this leads you in the right direction